app/src/androidTest/java/com/pajato/android/gamechat/TTTFragmentTest.java

```diff
@@ -44,13 +45,10 @@ public void navigateToGameFragment() throws InterruptedException {
         oValue = mRule.getActivity().getString(R.string.oValue);
         spaceValue = mRule.getActivity().getString(R.string.spaceValue);
 
-        onView(withId(R.id.chat_pane))
-                .perform(swipeLeft());
+        onView(withId(R.id.toolbar_game_icon))
+                .perform(click());
         onView(withId(R.id.board))
                 .check(matches(isDisplayed()));
-
-        // Very short sleep is required to mitigate a race condition between the tester and the code
-        Thread.sleep(250);
     }
 
     /**
```

Comment:
And conciseness is also why I have embraced the javadoc one-liner: /** Some grammatically correct and helpful comment. */